src/components/GalleryViewThumbnail.js

```diff
@@ -119,7 +123,6 @@ export class GalleryViewThumbnail extends Component {
 
 GalleryViewThumbnail.propTypes = {
   annotationsCount: PropTypes.number,
-  annotationSelected: PropTypes.bool,
```

Comment:
Context: We do not support selecting multiple annotations as of #3102. This prop is no longer needed.